src/GraphQL.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace Netzstrategen\WooCommerceLocalStore;
+
+/**
+ * GraphQL related functionalities.
+ */
+class GraphQL {
+
+  /**
+   * @implements graphql_register_types
+   */
+  public static function graphql_register_types() {
+    self::registerLocalStockStatusEnum();
+    self::registerLocalStockObjectType();
+    self::registerLocalStockField('product');
+    self::registerLocalStockField('ProductVariation');
+  }
+
+  /**
+   * Registers local stock status enum.
+   */
+  public static function registerLocalStockStatusEnum() {
+    $values = [
+      'HIGH' => ['value' => 'high'],
+      'LOW' => ['value' => 'low'],
+      'NONE' => ['value' => 'none'],
```

Comment:
Where are these uppercase values coming from?  The plugin only emits lowercase values:
https://github.com/netzstrategen/woocommerce-local-store/blob/4c4c5fd9d537a95bbde53ce3831595222b61cf6f/src/Product.php#L192